taraxa/state/dpos/precompiled/dpos_contract.go

```diff
@@ -187,10 +196,61 @@ func (self *Contract) RequiredGas(ctx vm.CallFrame, evm *vm.EVM) uint64 {
 	case "getValidatorEligibleVotesCount":
 		return DposGetMethodsGas
 	case "getValidators":
-	case "getDelegatorDelegations":
+		validators_count := uint64(self.validators.GetValidatorsCount())
```

Comment:
@MatusKysel do you know if "RequiredGas" method called only through dry_runner ? Not sure if we should not call here lazy_init so self.validators are initialized

Reply:
fixed